sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+| [`DO`](/sql-statements/sql-statement-do.md) | Executes an expression. |
+| [`EXPLAIN`](/sql-statements/sql-statement-explain.md) | Shows the execution plan of a query. |
+| [`TRACE`](/sql-statements/sql-statement-trace.md) | Traces query execution. |
+| [`USE`](/sql-statements/sql-statement-use.md) | Sets the current database. |
+
+## Show statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`SHOW BUILTINS`](/sql-statements/sql-statement-show-builtins.md) | Lists builtin functions. |
+| [`SHOW CHARACTER SET`](/sql-statements/sql-statement-show-character-set.md) | Lists character sets. |
+| [`SHOW COLLATIONS`](/sql-statements/sql-statement-show-collation.md) | Lists collations. |
+| [`SHOW ERRORS`](/sql-statements/sql-statement-show-errors.md) | Shows error messages. |
+| [`SHOW STATUS`](/sql-statements/sql-statement-show-status.md) | Shows server status. |
+| [`SHOW VARIABLES`](/sql-statements/sql-statement-show-variables.md) | Shows system variables. |
+| [`SHOW WARNINGS`](/sql-statements/sql-statement-show-warnings.md) | Shows warning messages. |
+
+## Instance management
+
+<CustomContent platform="tidb">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER INSTANCE`](/sql-statements/sql-statement-alter-instance.md) | Modifies an instance. |
+| [`FLUSH STATUS`](/sql-statements/sql-statement-flush-status.md) | Resets status variables. |
+| [`KILL`](/sql-statements/sql-statement-kill.md) | Kills a process. |
+| [`SHOW CONFIG`](/sql-statements/sql-statement-show-config.md) | Shows the configuration. |
+| [`SHOW ENGINES`](/sql-statements/sql-statement-show-engines.md) | Shows available storage engines. |
+| [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md) | Shows installed plugins. |
+| [`SHOW PROCESSLIST`](/sql-statements/sql-statement-show-processlist.md) | Shows active threads. |
+| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Shows query profiles. |
+| [`SHUTDOWN`](/sql-statements/sql-statement-shutdown.md) | Shuts down the database server. |
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER INSTANCE`](/sql-statements/sql-statement-alter-instance.md) | Modifies an instance. |
+| [`FLUSH STATUS`](/sql-statements/sql-statement-flush-status.md) | Resets status variables. |
+| [`KILL`](/sql-statements/sql-statement-kill.md) | Kills a process. |
+| [`SHOW ENGINES`](/sql-statements/sql-statement-show-engines.md) | Shows available storage engines. |
+| [`SHOW MASTER STATUS`](/sql-statements/sql-statement-show-master-status.md) | Shows the status of the master server. |
+| [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md) | Shows installed plugins. |
+| [`SHOW PROCESSLIST`](/sql-statements/sql-statement-show-processlist.md) | Shows active threads. |
+| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Shows query profiles. |
```

Comment:
```suggestion
| [`FLUSH STATUS`](/sql-statements/sql-statement-flush-status.md) | Included for compatibility with MySQL. TiDB uses Prometheus and Grafana for centralized metrics collection instead of `SHOW STATUS` for most metrics. |
| [`KILL`](/sql-statements/sql-statement-kill.md) | Kills a connection in any TiDB instance in the current TiDB cluster. |
| [`SHOW ENGINES`](/sql-statements/sql-statement-show-engines.md) | Shows available storage engines. |
| [`SHOW MASTER STATUS`](/sql-statements/sql-statement-show-master-status.md) | Shows the latest TSO in the cluster. |
| [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md) | Shows installed plugins. |
| [`SHOW PROCESSLIST`](/sql-statements/sql-statement-show-processlist.md) | Shows the current sessions connected to the same TiDB server. |
| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Currently only returns an empty result. |
```